services/web/server/src/simcore_service_webserver/projects/projects_api.py

```diff
@@ -1047,13 +1047,17 @@ async def patch_project_node(
     if _node_patch_exclude_unset.get("label"):
         await dynamic_scheduler_api.update_projects_networks(app, project_id=project_id)
 
-    # 5. Updates project states for user, if inputs have been changed
-    if "inputs" in _node_patch_exclude_unset:
+    # 5. Updates project states for user, if inputs/outputs have been changed
+    if {"inputs", "outputs"} & _node_patch_exclude_unset.keys():
```

Comment:
Cool, thanks! Maybe we can extend this test with these use cases?
`osparc-simcore/services/web/server/tests/unit/with_dbs/02/test_projects_nodes_handlers__patch.py`

Reply:
i agree with @matusdrobuliak66 . A test is necessary here. Either extend or a new one.